README.md

```diff
@@ -14,10 +14,63 @@ For installation instructions, please see our [install manual][install]
 
 ## Set up your Dev environment
 
-1. Make sure you have Postgresql installed
-1. Clone this repo
-1. Run `bin/setup`
-1. Start the app with `foreman start`
```

Comment:
Reason this was all 1's is that Markdown does a smart rendering of this, and you don't need to change all the numbers if you add something in between. Don't feel strong about this, but just so you know the old way also worked :-)

Reply:
@jvanbaarsen Ah! Yeah makes sense too. My stance on this is that a text file should be readable as text file too. (since that's what you do when you check out the repo and not have a Markdown viewer). But I understand this reasoning and can revert right away.

Reply:
@michiels No need to revert :-) I just wanted to make sure you knew that this was on purpose :D